packages/web/src/app/doc-pages/brand/icon-doc/icon-preview-filter/icon-preview-filter.component.html

```diff
@@ -0,0 +1,59 @@
+<div class="e-form-field">
+  <label class="e-form-field__label" for="searchIcons">Search</label>
+  <div class="e-input">
+    <i class="e-icon e-icon--search-color" aria-hidden="true"></i>
+    <input id="searchIcons" type="text" placeholder="Search through all icons" [(ngModel)]="term" />
+    @if (term().length > 0) {
+      <button class="e-btn e-btn--icon" (click)="term.set('')" aria-label="Empty search field">
+        <span class="e-btn__icon">
+          <i class="e-icon e-icon--close" aria-hidden="true" aria-hidden="true"></i>
+        </span>
+      </button>
+    }
+  </div>
+</div>
+
+<div class="filters e-none-sm">
+  <elvia-radio-filter
+    [items]="[
+      { label: 'All', value: 'all' },
+      { label: 'Outline', value: 'outline' },
+      { label: 'Filled', value: 'filled' },
+      { label: 'Two colored', value: 'colored' }
+    ]"
+    [value]="filter()"
+    [name]="'iconRadioFilters'"
+    [inlineStyle]="{ flexWrap: 'noWrap' }"
+    (valueOnChange)="filter.set($any($event).detail.value)"
+  />
+  <app-local-theme-switch [(theme)]="theme" />
+</div>
+
+<elvia-popover *ifViewportSize="['sm']">
+  <div slot="content">
+    <div class="e-mb-16">
+      <span class="e-text-md-strong">Filter</span>
+      <elvia-radio-filter
+        [items]="[
+          { label: 'All', value: 'all' },
+          { label: 'Outline', value: 'outline' },
+          { label: 'Filled', value: 'filled' },
+          { label: 'Two colored', value: 'colored' }
+        ]"
```

Comment:
Maybe we should define the items in the component file so that we can reuse them in both radio filters